CONTRIBUTING.md

```
@@ -189,8 +189,57 @@
+
+   ```bash
+   git checkout stage
+   git pull origin stage
+   git merge <branch-name>
+   ```
+
+   If you create a pull request to `stage`, your branch will be rebased. This can cause problems later when you try to create a pull request to `main`, as your branch will now include not only your changes but also those from others who have pushed to `stage`. To avoid this, use the commands above. This approach creates a merge commit on `stage` rather than rebasing your branch, keeping your changes isolated and the history cleaner.
+   > [!NOTE]
+   > Your changes aren't in `Staging` right away. CI/CD Jobs will create a deployment job that needs to be reviewed by your Project Manager / Team Lead first
+5. **Validate your changes in `Staging`**
+   
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 3

(MD009, no-trailing-spaces)